kernel/src/analysis/tree_sitter.rs

```diff
@@ -1,10 +1,10 @@
 use crate::model::analysis::{MatchNode, MatchNodeContext, TreeSitterNode};
 use crate::model::common::{Language, Position};
-use anyhow::{anyhow, Result};
+use anyhow::Result;
 use std::collections::HashMap;
 use tree_sitter::QueryCursor;
 
-fn get_tree_sitter_language(language: &Language) -> Option<tree_sitter::Language> {
+fn get_tree_sitter_language(language: &Language) -> tree_sitter::Language {
```

Comment:
Here, there's no need to return the `Option` at all, given that the code is always going to return a `Some` variant. This is introducing unnecessary complexity.

Reply:
More context [here](https://rust-lang.github.io/rust-clippy/master/index.html#/unnecessary_wraps)